frontend/src/components/RestaurantCard/RestaurantCard.tsx

```diff
@@ -0,0 +1,102 @@
+import { styled } from 'styled-components';
+import Label from '~/components/@common/Label';
+import { BORDER_RADIUS, FONT_SIZE } from '~/styles/common';
+import Star from '~/assets/icons/star.svg';
+
+interface RestaurantCardProps {
+  imageUrl: string;
+  name: string;
+  address: string;
+  category: string;
+  rating: string;
+  reviewCount: number;
+  isAds: boolean;
+}
+
+function RestaurantCard({ imageUrl, name, address, category, rating, reviewCount, isAds }: RestaurantCardProps) {
+  return (
+    <StyledDiv>
```

Comment:
최상단 Styled Components는 네이밍을 Container 정도로 가져가 주는 것이 어떨까요?
```suggestion
    <StyledContainer>
```